PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,119 @@
+- [ ] Are LED's located at useful places (comm, power, debugging, extra GPIO)?  
+- [ ] Are parts chosen easy to collect?  
+- [ ] Are parts chosen easy to solder?  
+- [ ] Is there reverse polarity protection on inputs?  
+- [ ] Do parts in the schematic match the actual part datasheet?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## BOM Requirements
+- [ ] **Bill Of Materials**: is there an Excel Sheet BOM and a Mouser Cart link?
+- [ ] Are *all* the correct parts in the BOM?
+- [ ] Is each part in the Mouser Cart in stock (or do we have the part in stock)?
+- [ ] Does each part in the Mouser Cart have a margin for part quantity?
+- [ ] Does every component in the schematic have a footprint, mouser link/datasheet, and reference number associated with it?
+
+**Please *read* every word on *every* bullet point before checking off the corresponding box.**
+
+## Layout Level Requirements 
+### 2D Spacing
+- [ ] The components are spaced out at an optimal distance.
+    - All components can be easily hand-soldered.
+    - IPC-SM-782A Standard requires a minimum distance of 1.0mm from edge to edge.
+- [ ] Components that are in parallel with each other are spaced out at an equal distance when possible.
+- [ ] The components are aligned with each other when possible.
+- [ ] Components are grouped based off of functionality.
+    - E.g. all components for CAN should be grouped.
+- [ ] Bypass capacitors are less than 1cm away from their respective IC's power pins.
+
+### 3D Spacing
+- [ ] Layout of components have been placed with mounting location and usage of the board in mind.
+    - Are PCBs going to be stacked on above/below this board? Are tall components placed accordingly?
+    - Are buttons and lights easily accessible and viewable?
+    - When mounted into the car, are the heights of the components and connectors accounted for?
+- [ ] I am not reading every word and I need to start this checklist over.
+    - Please uncheck everything you have checked and start over.
+- [ ] Location of connectors and wires going out of the board will prevent messy wiring.
+    - Are all the wires that are going in the same direction placed on the same edge of the board?
+
+### Components
+- [ ] Do footprints match the schematic?
+    - Make sure pins on the footprint correspond to pins in the schematic 
+    - (e.g. counterclockwise pins on footprint and alternating pins on schematic is *bad*)
+- [ ] Custom footprints have been double checked with the datasheet.
+- [ ] Footprints are consistent across the PCB: *i.e. all 0805 for Resistors/Capacitors*
+- [ ] Pin 1 of the footprint is labeled in some way. 
+- [ ] Are LED's in easy to see places? 
+- [ ] Are test points in easy to reach places?  
+- [ ] Are critical paths of switching converters as small as possible?  
+
+### Copper Layer 
+- [ ] The trace widths and trace clearances are greater than JLCPCB's minimum requirements. 
```

Comment:
Link to capabilities: https://jlcpcb.com/capabilities/Capabilities

Reply:
Perhaps agree upon and provide a constraints file
![image](https://user-images.githubusercontent.com/31521956/236361200-a5939886-36d3-4d66-8e44-687e1bf339b5.png)

Reply:
i did wanna do this. just forgot about it tbh.